src/cljs/webchange/dashboard/classes/class_modal/views.cljs

```diff
@@ -42,13 +42,14 @@
        (first)
        (:name)))
 
-(defn course-selector [props]
+(defn course-selector
+  [props handle-course-change]
```

Comment:
`handle-course-change` should be a part of `props`

Reply:
Done